### ISSUE TITLE
Load from HTTP(S)

### ISSUE THREAD

**Issue by thu-traub** [tillahoffmann/obsidian-file-include#8]
Would it be possible to load the file from and URL via HTTP?
If the name starts with "http?:" using fetch API?

**Comment by tillahoffmann** [comment]
Yes, I imagine that should be possible by modifying the code here. Contributions welcome!

https://github.com/tillahoffmann/obsidian-file-include/blob/cfc032eee40728c5e6d209922832b6a7739ce23d/main.ts#L20-L29

**Comment by thu-traub** [comment]
Ok, thanks
Was easy. I have a first version working.

How can I contribute?

Stefan

Von: Till Hoffmann ***@***.***>
Gesendet: Thursday, 1 August 2024 16:52
An: tillahoffmann/obsidian-file-include ***@***.***>
Cc: Traub, Stefan ***@***.***>; Author ***@***.***>
Betreff: Re: [tillahoffmann/obsidian-file-include] Load from HTTP(S) (Issue #8)


Yes, I imagine that should be possible by modifying the code here. Contributions welcome!

https://github.com/tillahoffmann/obsidian-file-include/blob/cfc032eee40728c5e6d209922832b6a7739ce23d/main.ts#L20-L29

**Comment by tillahoffmann** [comment]
Great! A pull request would work well.

**Comment by thu-traub** [comment]
I’ve created a pull request.
I hope I did this correct.

Von: Till Hoffmann ***@***.***>
Gesendet: Tuesday, 6 August 2024 17:49
An: tillahoffmann/obsidian-file-include ***@***.***>
Cc: Traub, Stefan ***@***.***>; Author ***@***.***>
Betreff: Re: [tillahoffmann/obsidian-file-include] Load from HTTP(S) (Issue #8)


Great! A pull request would work well.
